staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured_list.go

```diff
@@ -53,19 +53,22 @@ func (u *UnstructuredList) EachListItem(fn func(runtime.Object) error) error {
 }
 
 // UnstructuredContent returns a map contain an overlay of the Items field onto
-// the Object field. Items always overwrites overlay. Changing "items" in the
-// returned object will affect items in the underlying Items field, but changing
-// the "items" slice itself will have no effect.
-// TODO: expose SetUnstructuredContent on runtime.Unstructured that allows
-// items to be changed.
+// the Object field. Items always overwrites overlay.
 func (u *UnstructuredList) UnstructuredContent() map[string]interface{} {
-	out := u.Object
-	if out == nil {
-		out = make(map[string]interface{})
+	out := make(map[string]interface{})
```

Comment:
Pre-allocate the map? `make(map[string]interface{}, len(u.Object)+1)`. `+1` to avoid re-allocation in the case when `items` is not in the map.